test/integration/dnet/multi.bats

```diff
@@ -22,6 +22,7 @@ function is_network_exist() {
 	run dnet_cmd $(inst_id2port $i) network create -d test ${oname}
 	echo ${output}
 	[ "$status" -eq 0 ]
+	done
```

Comment:
This is not correct. The intention of this test is to create a network from one node and check if that network can be seen from all the nodes and then try to delete the network from the 2nd node always (irrespective of where it was created)